### PR TITLE
Added services

### PR DESCRIPTION
**Pull request by nishasen**
Added services to make api calls to add and retrieve data from firebase

### CODE REVIEW

File: social/src/Services/getAllPost.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        querySnapshot.forEach((doc) => {
+            posts.push({postId: doc.id, data: doc.data()});
+        })
+        return posts;
+    } catch(error) {
+        console.log(error)
+    }
+})
+
+//, where("privacy", "==", "public")
```
Comment (sadathshariff) [inline social/src/Services/getAllPost.js:19]: remove the commented code here

**Review by sadathshariff — APPROVED** [review]
Everything looks clean and nicely  broken down the functions and structured, like how you are handling the loading state

File: social/src/Services/UpdateUser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { db } from '../firebase';
+import { doc, updateDoc } from "firebase/firestore";
+import { showToast, isLoading } from '../Features';
+import { getUser } from '.';
+
+export const UpdateUser = async(form, dispatch, token) => {                          
+    dispatch(isLoading(true))
+  try {
+    await updateDoc(doc(db, "users", token), form);
```
Comment (kjais1720) [inline social/src/Services/UpdateUser.js:9]: use proper variable name for the `form` variable like, `updatedUserData` etc.

**Review by kjais1720 — APPROVED** [review]

File: social/src/Services/UpdateUser.js (hunk 2 above)
Comment (nishasen) [reply social/src/Services/UpdateUser.js:9]: Okay will change this

**Pull request merged**